app/person/tests.py

```diff
@@ -32,3 +39,23 @@ def test_name_label(self):
     person = Person.objects.get(id=1)
     label = person._meta.get_field('name').verbose_name
     self.assertEqual(label, 'Full name')
+  
+  def test_update_screens_button_renders(self):
+    User.objects.create_superuser(test_username, test_email, test_password)
```

Comment:
These user creation tasks could be pulled up into the setUpTestData function.

Reply:
Ah good point - this was updated in b490133.